frontend/test/metabase/scenarios/question/notebook.cy.spec.js

```diff
@@ -70,6 +70,16 @@ describe("scenarios > question > notebook", () => {
     cy.contains("Showing 1 row"); // ensure only one user was returned
   });
 
+  it("shouldn't show sub-dimensions for FK (metabase#16787)", () => {
+    openOrdersTable({ mode: "notebook" });
+    cy.findByText("Summarize").click();
+    cy.findByText("Pick a column to group by").click();
+    cy.findByText("User ID")
+      .closest(".List-item")
+      .find(".Field-extra")
+      .should("not.have.descendants", "*");
```

Comment:
Oh, I like this very much! Elegant solution 👌
FWIW, we shouldn't even display the `.Field-extra` as an empty div, but that's out of the scope of this PR.

Great work, @pawit-metabase !